book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative('./rental')
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental) unless @rentals.include?(rental)
```

Comment:
- [ ] [OPTIONAL] Kindly remove the line for adding the new Rental to the `@rentals` as you already add it in the Rental class initialize method. This will be marked as optional as you checked if the new rental exists in the `@rentals` list before or not, so it will not affect your code and it will be added once to the `@rentals` list.

Reply:
Ok, thanks for the clarification.